includes/wc-payment-api/class-wc-payments-http.php

```diff
@@ -49,7 +49,7 @@ public function init_hooks() {
 	 * @param array  $args             - The arguments to passed to Jetpack.
 	 * @param string $body             - The body passed on to the HTTP request.
 	 * @param bool   $is_site_specific - If true, the site ID will be included in the request url. Defaults to true.
-	 * @param bool   $use_user_token   - If true, the request will be signed with the user token rather than blog token. Defaults to false.
+	 * @param bool   $use_user_token   - If true, the request will be signed with the Jetpack connection owner user token rather than blog token. Defaults to false.
```

Comment:
I've added this distinction here, which would have helped me resolve this issue.

Reply:
Should we change the name of this param? `user_token` seems misleading.